cmd/token.go

```diff
@@ -0,0 +1,192 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/compute/metadata"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/google/go-tpm-tools/client"
+	"github.com/google/go-tpm-tools/launcher/agent"
```

Comment:
This PR encodes a dependency on the `launcher` submodule. We should separate out the `agent` logic if it will be used between the two.